Lib/test/test_socket.py

```diff
@@ -2562,8 +2562,7 @@ def testHyperVConstants(self):
         socket.HV_GUID_LOOPBACK
 
     def testCreateHyperVSocketWithUnknownProtoFailure(self):
-        expected = "A protocol was specified in the socket function call " \
-            "that does not support the semantics of the socket type requested"
+        expected = "[WinError 10041]"
```

Comment:
```suggestion
        expected = r"\[WinError 10041\]"
```

This is a regex, right? So we need to escape the square brackets.